docs/source/private-data/private-data.md

```diff
@@ -151,6 +151,15 @@ documentation on [transaction flow](../txflow.html).
    their copy of the private state database and private writeset storage. The
    private data is then deleted from the `transient data store`.
 
+[Fabric Gateway](../gateway.html) collects the required set of endorsements
```

Comment:
Andy, I was thinking we'd update steps 1 through 5 above to indicate that gateway orchestrates the flow, rather than client application. That's what I did for https://hyperledger-fabric.readthedocs.io/en/latest/txflow.html topic. I called it the "target peer", but maybe we should change that to "target gateway peer".

Alternatively, in each of these flow descriptions we could leave it open ended with language like "Client application (or gateway peer) submits proposal to endorsing peers", and describe at the beginning that you can either have the application orchestrate the flow if using a legacy SDK, or have the gateway peer orchestrate the flow if using a gateway SDK.

What do you think @joshhus ? You've got the same dilemma in the peer topic.

Reply:
target peer is good ... we are avoiding "gateway peer" usages in favor of "gateway service" (running on the target peer).